src/Symfony/Bridge/ProxyManager/Tests/LazyProxy/PhpDumper/Fixtures/proxy-implem.php

```diff
@@ -200,7 +200,7 @@ public function __wakeup()
     {
     }
 
-    public function setProxyInitializer(\Closure $initializer = null)%S
+    public function setProxyInitializer(%S\Closure $initializer = null)%S
```

Comment:
Why can't we do this?

```suggestion
    public function setProxyInitializer(?\Closure $initializer = null)%S
```

Reply:
because we support many versions of proxy-manager and only the latest version generates the `?`:
https://twitter.com/nicolasgrekas/status/1770016522599079996

Reply:
Makes sense, thank you.